x/dispensation/module.go

```diff
@@ -160,7 +160,8 @@ func (am AppModule) ExportGenesis(ctx sdk.Context, codec codec.JSONCodec) json.R
 }
 
 // BeginBlock returns the begin blocker for the dispensation module.
-func (am AppModule) BeginBlock(_ sdk.Context, _ abci.RequestBeginBlock) {
+func (am AppModule) BeginBlock(ctx sdk.Context, _ abci.RequestBeginBlock) {
+	BeginBlocker(ctx, am.keeper)
```

Comment:
Could do with a test for mint amounts and maximums etc.

Reply:
Sure , ill add that

Reply:
http://github.com/Sifchain/sifnode/blob/650839ad5a10c8164e68e9cb066d2abc65866d4e/x/dispensation/abci_test.go#L14-L61